app/views/responsible_body/donated_devices/interest/about.html.erb

```diff
@@ -0,0 +1,22 @@
+<%- title = t('page_titles.donated_devices.responsible_body.interest.about.title') %>
+<%- content_for :title, title %>
+<%- content_for :before_content, govuk_link_to('Back', responsible_body_home_path, class: 'govuk-back-link') %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl"><%= title %></h1>
+
+    <h2 class="govuk-heading-m">Devices can vary</h2>
+    <p class="govuk-body">A variety of devices are being donated. The devices a school or college could get might not all be the same.</p>
+
+    <p class="govuk-body">Every device will meet the <%= govuk_link_to 'minimum requirements needed for remote learning', 'https://www.computacenter.com/uk/daily-mail', target: '_blank', rel: 'noopener noreferrer' %>.</p>
+    
+    <h2 class="govuk-heading-m">Configure at your own cost</h2>
+    <p class="govuk-body">Devices are provided with factory settings, schools and colleges will need to configure them at their own cost before lending them to children and young people.</p>
+    <p class="govuk-body govuk-!-margin-bottom-7">Schools and colleges will need to purchase licences for any Google Chromebooks they receive.</p>    
```

Comment:
This seems a little odd - I would have expected to purchase licenses for any Windows laptops I receive, but not for Chromebooks. Are we sure that's right?